resource/ui/ts/util/convert.ts

```diff
@@ -1,15 +1,25 @@
 // source: util/convert.ts
 // Author: Matt Tracy (matt@cockroachlabs.com)
+// Author: Bram Gruneir (bram+code@cockroachlabs.com)
 
 /**
  * Utils contains common utilities.
  */
 module Utils {
-    /**
-     * microToNanos is intended to convert milliseconds (used by Java time
-     * methods) to nanoseconds, which are used by the cockroach server.
-     */
-    export function milliToNanos(millis:number):number {
-        return millis*1.0e6;
-    }
+	export module Convert {
+		/**
+		 * microToNanos is intended to convert milliseconds (used by Java time
+		 * methods) to nanoseconds, which are used by the cockroach server.
+		 */
+		export function MilliToNanos(millis: number): number {
+			return millis * 1.0e6;
+		}
+
+		/**
+		 * TimestampToDate converts a timestamp (nanos) into a Date.
+		 */
+		export function TimestampToDate(timestamp: number): Date {
+			return new Date(timestamp / 1.0e6);
```

Comment:
There's little indication other than the comment that the timestamp is in nanoseconds.

It's well known in Javascript that Date() accepts a millisecond unix timestamp.  Therefore, I think what you really want is `NanosToMilli`, and then in places where you are already using `TimestampToDate`, you really want to use `new Date(Convert.NanosToMilli(timestamp))`